bin/print-chrome-path.js

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env node
```

Comment:
I assume this what chmod +x'd, although I don't know if there is a way to confirm that in GH ui.

related, benefits to using shebang vs just setting the bin script to `node bin/print-chrome-path.js` ?

Reply:
> I assume this was `chmod +x`'d, although I don't know if there is a way to confirm that in GH ui

npm/yarn does the chmod on install

> related, benefits to using shebang vs just setting the bin script to `node bin/print-chrome-path.js` ?

unlike `script` entries, `bin` entries are just filenames that are symlinked into `node_modules/.bin`, so need the shebang

Reply:
TIL, thx!